### PR TITLE
chore: update cli to 1.10.0-next-1616100617000

### PR DESCRIPTION
**Pull request by yury-s**
* Also add bots for testing against stable channels of Chrome and Edge.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -48,3 +48,41 @@ jobs:
...
+        include:
+          - os: windows-latest
+            browser-channel: msedge
+          - os: macos-latest
+            browser-channel: msedge
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: microsoft/playwright-github-action@v1
+      - name: Set up JDK 1.8
```
Comment (pavelfeldman) [inline .github/workflows/test.yml:69]: - name: Install Media Pack
      if: matrix.os == 'windows-latest'
      shell: powershell
      run: Install-WindowsFeature Server-Media-Foundation

**Review by pavelfeldman — APPROVED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (yury-s) [reply .github/workflows/test.yml:69]: done!
